### ISSUE TITLE
cAdvisor serving publicly

### ISSUE THREAD

**Issue by davidhiendl** [rancher/rancher#7668]
**Rancher Versions:**
Server: v1.3.3
healthcheck: v0.2.0
ipsec:  0.0.2
network-services: v0.0.8
scheduler:
kubernetes (if applicable):  v1.5.1-rancher1-8

**Docker Version:**
docker --version
Docker version 1.12.6, build 78d1802

**OS and where are the hosts located? (cloud, bare metal, etc):**
Linode VMs / Bare-Metal

**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
Single Node Rancher with public IP and cert

**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**
Kubernetes

**Steps to Reproduce:**
1. Setup Rancher
2. Setup Plain Kubernetes Cluster (3 hosts in my example), no pods/services needed
3. Check Kubernetes Nodes public/private IPs for metrics on 
- http://node-ip:10255/metrics
- http://node-ip:4194/containers

**Results:** 
Metrics are publicly visible to anyone.

**Expected:**
Metrics exposed after authentication or only inside the overlay network.

See: https://github.com/kubernetes/kubernetes/issues/11710
Basically they do set the cadvisor port to 0. Might be worth looking into.

**Comment by superseb** [comment]
Merging into https://github.com/rancher/rancher/issues/12142
